addons/website/controllers/form.py

```diff
@@ -173,6 +173,10 @@ def extract_data(self, model, values):
                 except ValueError:
                     error_fields.append(field_name)
 
+                if dest_model._name == 'mail.mail' and field_name == 'email_from':
+                    # Always add the email in the custom message
```

Comment:
Could we elaborate a bit?
Something like:

> As the "email_from" is used to populate the email_from of the sent mail.mail,
> it could be filtered out at sending time (since #[reference-to-PR]).
> To make sure the email contains that (important) information, we also add it
> to the "custom message" that will be included in the body of the email sent.

(If I understood the issue properly).